.github/workflows/tests.yml

```
@@ -23,8 +23,11 @@ jobs:
       - name: Install Dependencies (Server)
         working-directory: ./server
         run: bun install
+      - name: Install Dependencies (Frontend)
+        working-directory: ./frontend
+        run: bun install
       - name: Run tests
-        run: bun test
+        run: bun run test 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)